docker-compose.yml

```diff
@@ -70,3 +70,13 @@ services:
         RAILS_VERSION: 4.2
     volumes:
       - .:/source:delegated
+
+  ruby-2.3-rails-4.1:
+    build:
+      context: .
+      dockerfile: dockerfiles/ruby23.dockerfile
+      args:
+        FROM: ruby:2.3
```

Comment:
We don't need this because we always use `ruby:2.3` for this image.

Reply:
Done.